azure-functions/acbs-function/activity-create-deal-investor/index.js

```diff
@@ -55,8 +55,8 @@ const createDealInvestor = async (context) => {
       ...data,
     };
   } catch (error) {
-    console.error('Unable to create deal investor record.', { error });
-    throw new Error(error);
+    console.error('Unable to create deal investor record. %O', error);
+    throw new Error('Unable to create deal investor record.');
```

Comment:
Some error logs and `new Error`'s have full stops, some don't. Should they be consistent?

Reply:
done.  Removed all full stops